magprime/templates/regextra.html

```diff
@@ -9,9 +9,14 @@
         }
         $("input:radio[name='amount_extra'][value=40]").parents('.btn').append($('#guitar_keychain'));
         $('#guitar_keychain').on('click', function(e) { e.stopPropagation(); })
-        if ($("input:radio[name='amount_extra'][value=250]")) {
+        if ($("input:radio[name='amount_extra'][value={{ c.SUPPORTER_LEVEL }}]")) {
```

Comment:
We should probably use other constants here if we're referring to these values in code.  The way that the ``[enums]`` section works is that you can define numeric values there and then use them later in the config, e.g. you can see what we're doing at https://github.com/magfest/ubersystem/blob/master/uber/configspec.ini#L393 with our ``shirt_level`` and ``supporter_level`` values.